src/Cli/StubLoop.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Queue\Cli;
+
+/**
+ * Stub loop.
```

Comment:
It's used in tests only, so let's move it to tests too. We've got tests/App directory for such stubs.

Reply:
Stub extracted to separate namespace for stubs